### PR TITLE
`imops._numeric`, `imops.measure.center_of_mass`, benchmarks

### PR DESCRIPTION
**Pull request by vovaf709**

### CODE REVIEW

File: imops/measure.py [hunk 1]
```diff
@@ -73,6 +85,8 @@ def label(
 
     if ndim > 3:
         warn("Fast label is only supported for ndim<=3, Falling back to scikit-image's implementation.")
+        if dtype is not None:
+            warn("Explicitly passed `dtype` has no effect if fallback to scikit-image's implementation has occurred.")
```
Comment (maxme1) [inline imops/measure.py:89]: this is bad. we should guarantee that the dtype the user requested will coincide with the one from the resulting array

File: imops/measure.py [hunk 2]
```diff
@@ -108,3 +126,45 @@ def label(
...
+    -------
+    center_of_mass: tuple, or list of tuples
+        coordinates of centers-of-mass
+
+    Examples
+    --------
+    >>> center = center_of_mass(np.ones((2, 2)))  # (0.5, 0.5)
+    """
+    normalizer = parallel_sum(array.ravel(), num_threads=num_threads, backend=backend)
+    grids = np.ogrid[[slice(0, i) for i in array.shape]]
```
Comment (maxme1) [inline imops/measure.py:155]: `list(map(slice, array.shape))`?

File: setup.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def __repr__(self):
 
 # Cython extension and .pyx source file names must be the same to compile
 # https://stackoverflow.com/questions/8024805/cython-compiled-c-extension-importerror-dynamic-module-does-not-define-init-fu
-modules = ['backprojection', 'radon', 'zoom', 'morphology']
+modules = ['backprojection', 'radon', 'zoom', 'morphology', 'numeric']
```
Comment (maxme1) [inline setup.py:39]: shouldn't it be _numeric?

File: _pyproject_build.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+        name = 'imops'
+        args = ['-fopenmp']
+
+        if self.distribution.ext_modules is None:
+            self.distribution.ext_modules = []
+
+        # Cython extension and .pyx source file names must be the same to compile
+        # https://stackoverflow.com/questions/8024805/cython-compiled-c-extension-importerror-dynamic-module-does-not-define-init-fu
+        modules = ['backprojection', 'radon', 'zoom', 'morphology', 'numeric']
```
Comment (maxme1) [inline _pyproject_build.py:33]: same with _numeric

File: docs/index.md [hunk 5]
```diff
@@ -43,6 +43,12 @@ pip install imops[numba]  # additionally install Numba backend
...
 
+::: imops.measure.center_of_mass
+
 ::: imops.radon.radon
 
 ::: imops.radon.inverse_radon
+
+::: imops._numeric._sum
+
+::: imops._numeric._mul
```
Comment (maxme1) [inline docs/index.md:54]: shouldn't include private functions and modules to public docs

File: setup.py (hunk 3 above)
Comment (vovaf709) [reply setup.py:39]: No, all `.pyx` files are underscored, see L42

File: docs/index.md (hunk 5 above)
Comment (vovaf709) [reply docs/index.md]: Same, see L39-43

File: _pyproject_build.py (hunk 4 above)
Comment (vovaf709) [reply _pyproject_build.py:33]: Same, see L39-43

**Pull request merged**